go.mod

```diff
@@ -1,9 +1,9 @@
-module asynqmon
+module github.com/hibiken/asynqmon
 
 go 1.16
 
 require (
-	github.com/go-redis/redis/v8 v8.8.0
```

Comment:
Is this intentional? I believe we want to use `go-redis/redis/v8`

Reply:
Back when I first tried asynq, on master it was v7 so I was having issues with using asynq and asynqmon together for using `redis.UniversalClient` in router options.

I will change this back to v8, now that asynq/master is also on v8.